import-service/serverless.ts

```diff
@@ -0,0 +1,51 @@
+import type { AWS } from "@serverless/typescript";
+
+import importProductsFile from "@functions/importProductsFile";
+import importFileParser from "@functions/importFileParser";
+
+const BUCKET = "stone-shop-import-service";
+
+const serverlessConfiguration: AWS = {
+  service: "import-service",
+  frameworkVersion: "2",
+  custom: {
+    webpack: {
+      webpackConfig: "./webpack.config.js",
+      includeModules: true,
+    },
+  },
+  plugins: ["serverless-webpack"],
+  provider: {
+    name: "aws",
+    runtime: "nodejs14.x",
+    region: "eu-west-1",
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: "1",
+    },
+    lambdaHashingVersion: "20201221",
+    iam: {
+      role: {
+        statements: [
+          {
+            Effect: "Allow",
+            Action: "s3:ListBucket",
+            Resource: [`arn:aws:s3:::${BUCKET}`],
+          },
+          {
+            Effect: "Allow",
+            Action: "s3:*",
+            Resource: [`arn:aws:s3:::${BUCKET}/*`],
+          },
+        ],
+      },
+    },
+  },
+  // import the function via paths
+  functions: { importProductsFile, importFileParser },
+};
```

Comment:
everything is good but if I deploy this without the bucket being created then I have to setup CORS for this bucket